Unable to test in safari?

``` sh
$ BROWSER=safari make test-browser
node_modules/.bin/duo-test browser safari \
      --reporter spec \
      --build tests.js \
      --port 5678 \
      --commands "make tests.js"

  Error: "safari" is not supported, supported browsers: firefox,chrome,safari,opera,ie
    at callee$0$0$ (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/bin/browser.js:39:10)
    at Empty.invoke (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/gnode/node_modules/regenerator/runtime.js:180:31)
    at next (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/co/index.js:74:21)
    at Command.<anonymous> (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/co/index.js:45:5)
    at next (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/co/index.js:90:21)
    at Command.<anonymous> (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/co/index.js:45:5)
    at Command.listener (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/commander/index.js:282:8)
    at Command.emit (events.js:98:17)
    at Command.parseArgs (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/commander/index.js:524:12)
    at Command.parse (/Users/stephenmathieson/repos/github.com/stephenmathieson/temporarily-insert/node_modules/duo-test/node_modules/commander/index.js:410:21)

make: *** [test-browser] Error 1
```

--- a/bin/browser.js
+++ b/bin/browser.js
@@ -37,9 +37,9 @@ module.exports = function*(cmd, dt, args){
 
 function name(name){
   switch (name) {
-    case ~browsers.indexOf(name): return name;
     case 'chrome': return 'google chrome';
     case 'ie': return 'internet explorer';
     case 'ff': return 'firefox';
   }
+  if (~browsers.indexOf(name)) return name;
 }

